src/ModelReflector.php

```diff
@@ -97,20 +97,26 @@ public function getModelRelations($model, bool $forceRefresh = false): Collectio
 
         foreach ($reflectionMethods as $reflectionMethod) {
             if ($this->reflectionMethodIsRelation($reflectionMethod)) {
-                $methodName   = $reflectionMethod->getName();
-                $relation     = $modelInstance->{$methodName}();
-                $relatedModel = $relation->getRelated();
-
+                $methodName                    = $reflectionMethod->getName();
```

Comment:
Why are there so many spaces? Should probably reformat once more.

Reply:
yes